src/mountutils.cpp

```diff
@@ -22,4 +24,12 @@ NAN_MODULE_INIT(MountUtilsInit) {
   .ToLocalChecked());
 }
 
+
+void MountUtilsLog(std::string string) {
```

Comment:
I'm not sure we would get an advantage out of them. Maybe lets try with this, and improve if we find a use case?